### PR TITLE
 Hong kong chinese

### PR DESCRIPTION
**Pull request by jackshallattack**
Made this for someone on discord asking to do it, tell me if i need to delete it

### CODE REVIEW

File: Hong kong chinese [hunk 1]
```diff
@@ -0,0 +1,777 @@
+module.exports = `
+menu.btn.ranked Ranked
```
Comment (DansivDan) [inline Hong kong chinese:2]: /段位

Comment (DansivDan) [inline Hong kong chinese:2]: 段位

**Comment by mty22** [comment]
Feel free to push a pull request once the translation is complete. This is just `en.js` renamed -> pushed.

File: Hong kong chinese [hunk 2]
```diff
@@ -0,0 +1,777 @@
...
+menu.btn.shop Shop
```
Comment (DansivDan) [inline Hong kong chinese:3]: 商店

File: Hong kong chinese [hunk 3]
```diff
@@ -0,0 +1,777 @@
...
+menu.btn.social Hub
```
Comment (DansivDan) [inline Hong kong chinese:4]: 社交中心

**Comment by mty22** [comment]
@DansivDan best to translate outside of github rather than line by line here.

**Pull request closed without merging**